docs/tools/tools-home.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Tools
+layout: default
+has_children: true
+nav_order: 6
+has_toc: true
+---
+
+# FeatureBase Tools
+{: .no_toc }
+
+FeatureBase tools...
```

Comment:
You can remove "Featurebase tools..." because it doesn't give any further meaning than the page title.

What would be useful would be to explain
* What the context of FeatureBase tools actually is
* Why a user would use FeatureBase tools rather than the UI
* Who FeatureBase tools are for
* When FeatureBase tools should/should not be used

In short, I'd say to remove the concept of "FeatureBase Tools" until we've got more than one. Because the "s" implies there's multiple tools and right now there's just FBSQL

Reply:
Plus with any new docs it's a good idea to include me or @greg in the review because we're ultimately responsible for help.

Here's an overall workflow for contiributing content to featurebase-docs:

https://github.com/FeatureBaseDB/featurebase-docs/tree/main/help-on-help/publishing-workflow